### DISCUSSION TITLE
Set parent with resave/entries

### DISCUSSION THREAD

**Discussion by pixelmachine** [craftcms/cms#11180] [Q&A] [answered]
Is it possible to set a parent with the `resave/entries` command? I can't get it to work successfully.

I tried a few options like:
```php
craft resave/entries --element-id 2728 --set parent --to "fn(\$element) => \$element->getSiblings()->id(37374)->one()"
```
It loops through the entries and saves them without error but there's no change to the parent setting. The IDs are correct and the getSiblings method returns the correct entry from the templates.

**Comment by brandonkelly** [comment, marked as answer]
Yeah it’s possible.

Craft 3:

```sh
php craft resave/entries --element-id 2728 --set newParentId --to "=37374"
```

Craft 4:

```sh
php craft resave/entries --element-id 2728 --set parentId --to "=37374"
```

**Reply by pixelmachine** [reply]
Ah thanks. I wasn't aware newParentId existed for entries, it's only listed on category models – I looked everywhere for some way to set the ID directly! 

Do you know why my function didn't work?

**Reply by brandonkelly** [reply]
It didn’t work because setting a new parent via a `parent` property is not supported.
